protocol/proofs/fdg-refund-mode.md

```diff
@@ -0,0 +1,95 @@
+incident response process. `FaultDisputeGame` stores player bonds inside of the `DelayedWETH`
+contract which imposes a 7 day delay on bond withdrawals. During this delay period, the 2/2 Safe
+managed jointly by the Security Council and the Optimism Foundation can hold bonds from specific
+games or from all games at the same time. Held bonds are transferred to this 2/2 Safe where they
+can be manually disbursed to the correct recipients.
+
+## Problem Statement
+
+Although `DelayedWETH` is an effective tool at preventing a buggy or malicious `FaultDisputeGame`
+from distributing bonds to a potential attacker, the actual process of returning those bonds back
+to the correct players is underspecified. Underspecified proceses can introduce significant stress
+to already stressful situations where executive decisions around fallbacks must be made quickly and
+confidently. We therefore want to introduce alternative low-stress options for bond recovery.
+
+## Proposed Solution
+
+We propose modifying the `FaultDisputeGame` to enter a "refund mode" if the game is blacklisted.
+When the `FaultDisputeGame` is in this refund mode it will distribute bonds back to the users who
+originally deposited those bonds.
+
+### Preventing Complex States
+
+`FaultDisputeGame` currently allows users to begin withdrawing bonds before the game has fully
+resolved. It would likely be necessary to modify the `FaultDisputeGame` to block withdrawals until
+full resolution so that we can more easily reason about the security properties of this change. It
+may otherwise be possible to be caught in strange states.
+
+For instance, `FaultDisputeGame` instances are usually expected to be blacklisted only after they
+resolve incorrectly, so it may be possible for some users to begin withdrawing *before* a game is
+blacklisted, then the game becomes blacklisted and those same users can withdraw their original
+bonds. We can avoid this sort of double-counting and generally simplify the security model of the
+contract by preventing bond withdrawals until the game has resolved.
+
+We also likely need to prevent `FaultDisputeGame` contracts from entering refund mode if any
+withdrawals have already started to avoid dealing with complex state. If any user is able to
+withdraw before refund mode is activated then we assume that all withdrawals could've been
+completed and there's no reason to activate refund mode for those contracts.
+
+### Enabling Blanket Refunds
+
+Existing blacklisting functionality targets specific `FaultDisputeGame` instances and cannot easily
+handle cases where many games need to be blacklisted. If we were to adopt DD#76 alongside
+[DD#77](https://github.com/ethereum-optimism/design-docs/pull/77) then we could use the proposed
+`gameCreationValidFrom` timestamp to act as an implicit blacklist of all games created before this
+timestamp. The `AnchorStateRegistry.isValidGame(...)` function would return `false` if the game is
```

Comment:
Add note for flag to not go into refund mode if withdrawals have already started

Reply:
Edit: actually, handle this in `AnchorStateRegistry`

Reply:
I think we're not handling this in `AnchorStateRegistry` anymore right? Then we will need to track if withdrawals have started in `DelayedWETH`. I suspect just tracking if someone did withdraw is the simplest thing rather than trying to track if someone could have.